### PR TITLE
Add embedFileRelative function.

### PR DESCRIPTION
**Pull request by my-name-is-lad**
This PR adds a handful shortcut to embed files with path relative to project root. That's it.

### CODE REVIEW

**Review by snoyberg — CHANGES_REQUESTED** [review]
Can you add a minor version bump, changelog entry, and `@since` comment on the new function?

**Comment by my-name-is-lad** [comment]
Done.

File: ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # ChangeLog for file-embed
 
+## 0.0.15.1
```
Comment (snoyberg) [inline ChangeLog.md:3]: A minor version bump would bring this to 0.0.16, necessary by the PVP since this adds a new identifier.

**Review by snoyberg — CHANGES_REQUESTED** [review]

File: ChangeLog.md (hunk 1 above)
Comment (my-name-is-lad) [reply ChangeLog.md]: Fixed

**Review by snoyberg — APPROVED** [review]

**Comment by snoyberg** [comment]
New release is now on Hackage. Thanks!

**Pull request merged**